src/utils/wallet.js

```diff
@@ -103,19 +104,7 @@ export class Wallet {
       return await this.getAccount(this.accountId).getAccessKeys()
    }
 
-   async connectWapm(wamp) {
-      try {
-         return await new Promise((resolve, reject) => {
-            wamp.onopen = session => resolve(session)
-            wamp.onclose = reason => reject(reason)
-            wamp.open()
-         });
-      } catch (error) {
-         console.error('Connection failure due to:', error)
-         return
-      }
-   }
-
+   
    async getTransactions(accountId = '') {
```

Comment:
@marcinbodnar  I meant that `getTransaction` should also be part of `explorer-api`.